CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
+## 8.17.1
+  - Provides a guidance in logs when plugin version mismatches with connected Elasticsearch `major.minor` version [#255](https://github.com/elastic/logstash-filter-elastic_integration/pull/255)
+  - Embeds Ingest Node components from Elasticsearch 8.17
+  - Compatible with Logstash 8.15+
```

Comment:
Given that 8.17.0 is already 8.15+ compatible should we repeat also in this patch version?

Reply:
We recently make plugin `major.minor` alignment with stack versions and I support repeating this _important baseline_ (helps users which ES version to connect to, and tells LS version compatibility) to make _very visible_ for users.